README.md

```diff
@@ -1,2 +1,55 @@
-# 2022-G20-Bali-summit
-2022 G20 Bali summit
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# First Capstone Project
+
+**The 2022 G20 Bali summit is the upcoming seventeenth meeting of Group of Twenty (G20), the summit scheduled to take place in Bali, Indonesia, in 2022.**
+
+> One paragraph statement about the project.
+> ![2022-01-05_03-43-21](https://user-images.githubusercontent.com/63915024/148134074-9eb1871b-d56f-45e8-b250-3ee850e735aa.png)
+
+
+
+
+
+
+
+## Built With
+
+- HTML
+- CSS (Used Flexbox and Grid tools)
+- JavaScript
+- Bootstrap(Framework)
+- Git & GitHub
+
+## Live Showroom
+
+[Live Demo Link](https://mukhammadrizooff.github.io/2022-G20-Bali-summit/)
+
+## Website description
+
+[Presentation video](https://www.loom.com/share/1517a93228224fb6a64ab6bd521d811d)
+
+## Author
+
+👤 Author1
+
+- GitHub: [@mukhammadrizooff](https://github.com/mukhammadrizooff)
+- LinkedIn: [LinkedIn](linkedin.com/in/mukhammadrizooff)
+
+## Cloning a repository
+- On GitHub.com, navigate to the main page of the repository. Above the list of files, click Download icon and Code.
+- Copy for git clone
+- Type git clone, and then paste the URL you copied earlier.
+`$ git clone https://github.com/mukhammadrizooff/Portfolio_Template.git`
+- Finally press Enter 
+
+## Acknowledgements
+Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).
```

Comment:
Thank you for making this change from the previous reviewer. 👍

Reply:
### **You are welcome!**